e2e-tests/wiremock/src/main/java/com/foo/wiremock/WireMockApplication.java

```diff
@@ -0,0 +1,15 @@
+package com.foo.wiremock;
+
+
+import static org.hamcrest.Matchers.is;
+import static io.restassured.RestAssured.given;
+
+public class WireMockApplication {
+
+    public String callApi(int port) {
+        return given()
+                .when()
+                .get("http://localhost:" + port +"/api/call")
+                .body().asString();
+    }
```

Comment:
should return something simple, like a string X.
then, here, should have a trivial check like `if(x.equals("42"))`, and return `OK` if so